homeassistant/components/spotify/media_player.py

```diff
@@ -117,18 +113,8 @@ def __init__(
         """Initialize."""
         super().__init__(coordinator)
         self.devices = device_coordinator
-
         self._attr_unique_id = user_id
```

Comment:
`user_id` is just handed in for the ID, what about moving this to the base entity as well?

Reply:
In this case I don't have a suffix. In other examples I expect to be using an entity description so I'd rather set it in the platforms itself, since the unique id format changes per platform

Reply:
> In this case I don't have a suffix

Which I guess would be fine as this is the main entity (name=None). So in this case hand in `None` as suffix and don't append anything.